pkg/gce-pd-csi-driver/node.go

```diff
@@ -39,7 +38,7 @@ type GCENodeServer struct {
 
 	// A map storing all volumes with ongoing operations so that additional operations
```

Comment:
update the comments after changing to volumeLocks?

Reply:
I'm not sure that a comment is actually even needed here, as what VolumeLocks does is explained in by common.VolumeLocks